frontend/src/features/myinfo/utils/extractPreviewValue.ts

```diff
@@ -0,0 +1,18 @@
+import { BasicField, MyInfoPrefilledFormField } from '~shared/types'
+
+type PrefilledMyInfoValue = string | { value: string }
+
+export const extractPreviewValue = ({
```

Comment:
nit: this one could or could not be a previewValue, i think this can continue to be called `extractPrefilledValue` haha

Reply:
sounds good!